src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -540,5 +541,20 @@ private CodeAction TryCreateCodeAction(Document document, AddImportFixData fixDa
 
             throw ExceptionUtilities.Unreachable;
         }
+
+        protected ITypeSymbol GetAwaitInfo(SemanticModel semanticModel, SyntaxNode node, ISyntaxFactsService syntaxFactsService, CancellationToken cancellationToken)
```

Comment:
does this need to be protected?

Reply:
Good call, looks like it doesn't 👍 

```suggestion
        private ITypeSymbol GetAwaitInfo(SemanticModel semanticModel, SyntaxNode node, ISyntaxFactsService syntaxFactsService, CancellationToken cancellationToken)
```

Reply:
Makes sense, private works fine